### PR TITLE
Configure routes file and write some controller and view

### PR DESCRIPTION
**Pull request by francksefu**
[Project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#project-requirements)
Important: do NOT use rails generate scaffold command. The point of this project is to show you how to set up controllers by yourself.
Take a look at the wireframes below. Make sure that your app handles the URLs used in the wireframes:
All users page wireframe Single user page wireframe All posts by a given user page wireframe Single post page wireframe

For each URL you should create:

A route.
An action in the correct controller.
A view file.
View files should include only placeholders written in plain HTML, e.g:

Here is a list of posts for a given user
Make sure that you added code to handle only the 4 required URLs.

[Make sure that when you run the rails routes command in your console, you can see the following output:](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#make-sure-that-when-you-run-the-rails-routes-command-in-your-console-you-can-see-the-following-output)
Single Post page wireframe

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class PostsController < ApplicationController
+  # Create action - Handle the creation of a new post
+
+  def create
+    @user = User.find(params[:author_id])
+    @post = @user.posts.create(post_params)
+    redirect_to user_path(@user)
+  end
+
```
Comment (julie-ify) [inline app/controllers/posts_controller.rb]: - [ ] You don't have any create route for posts, so this action is somewhat useless. And it is not part of this project. Kindly remove this controller action.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @francksefu,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @francksefu,

## APPROVED 🟢

## Highlights
✔️ Project met all technical requirements

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @francksefu 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


Highlights
- [x] Good project organization ✔️
- [x] Nice use of Rails features ✔️  
- [x] Kudus on meeting the requirements on routes paths ✔️


Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**